### PR TITLE
Fixed #1

### LINKED ISSUE #1: Add an icon for the extension
Add a suitable icon that represents the chrome extension.

### PR DESCRIPTION
**Pull request by ShivaniArun**
Added a Google icon with a red cross on it

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -8,5 +8,7 @@
     "scripts": ["dont-record-my-search.js"],
     "persistent": false
   },
-  "manifest_version": 2
```
Comment (aa-dit-yuh) [inline manifest.json:11]: Why did you delete this line of code?

Comment (ShivaniArun) [reply manifest.json:11]: The line already exists at line number 2.

Comment (aa-dit-yuh) [reply manifest.json:11]: Good enough

**Comment by aa-dit-yuh** [comment]
LGTM. Thanks.
